csrc/quantization/smoothquant/fused_kernels.cu

```diff
@@ -0,0 +1,91 @@
+#include <ATen/cuda/CUDAContext.h>
```

Comment:
Should this file be called something else?

Reply:
@varun-sundar-rabindranath

Reply:
Moved in https://github.com/neuralmagic/nm-vllm/pull/215